src/internal/account.ts

```diff
@@ -795,3 +803,104 @@ export async function isAccountExist(args: { aptosConfig: AptosConfig; authKey:
     throw new Error(`Error while looking for an account info ${accountAddress.toString()}`);
   }
 }
+
+/**
+ * Verifies a Secp256k1 account by checking the signature against the message and account's authentication key.
+ *
+ * This function takes a message and signature, and attempts to recover the public key that created the signature.
+ * It then verifies that the recovered public key matches the authentication key stored on-chain for the provided account address.
+ *
+ * If a recovery bit is provided, it will only attempt verification with that specific recovery bit.
+ * Otherwise, it will try all possible recovery bits (0-3) until it finds a match.
+ *
+ * @param args - The arguments for verifying the Secp256k1 account
+ * @param args.aptosConfig - The configuration for connecting to the Aptos blockchain.
+ * @param args.message - The message that was signed
+ * @param args.signature - The signature to verify (either raw hex or Secp256k1Signature object)
+ * @param args.recoveryBit - Optional specific recovery bit to use for verification
+ * @param args.accountAddress - The address of the account to verify
+ * @returns The recovered public key if verification succeeds
+ * @throws Error if verification fails or no matching public key is found
+ *
+ * @example
+ * ```typescript
+ * import { AptosConfig, Network } from "@aptos-labs/ts-sdk";
+ *
+ * const config = new AptosConfig({ network: Network.DEVNET });
+ *
+ * const publicKey = await verifySecp256k1Account({
+ *   aptosConfig,
+ *   message: "0x1234...",
+ *   signature: "0x5678...",
+ *   accountAddress: "0x1"
+ * });
+ * ```
+ */
+export async function verifySecp256k1Account(args: {
+  aptosConfig: AptosConfig;
+  message: HexInput;
+  signature: HexInput | Secp256k1Signature | AnySignature;
+  recoveryBit?: number;
+  accountAddress: AccountAddressInput;
+}): Promise<AnyPublicKey> {
```

Comment:
This function seems to solve a very specific use case:
obtain a public key for an account, from a signed message and address, only for Secp256k1 accounts.

Why would I need it as a dapp developer?
Are you using this internally somewhere?